JoonasHeinonen/escape-from-zordoom-prison
# feature/ladder

## Changes:
Added the Rivet player ability to climb a ladder

## How to test:
go to a ladder that is near the player as Rivet( please quit the game and pick rivet as the person you want to play)
press the w key to move up the ladder and the s key to go down.
Move up and down and then jump off the ladder.
See if the animations look ok or need improvement.
## Sources cited:
https://www.youtube.com/watch?v=njY3NFnAI8o
